### DISCUSSION TITLE
Account flagged

### DISCUSSION THREAD

**Discussion by ahmad6532** [community/community#80905] [New to GitHub]
My account has been flagged. When i push or pull the code the message " You are marked as spam, and therefore cannot authorize a third party application. " shows. How i can recover it

**Comment by queenofcorgis** [comment]
Hi @ahmad6532 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
